despre.html

```diff
@@ -74,13 +74,24 @@
 
       <nav class="main-nav">
         <ul class="main-nav-list">
-          <li><a class="main-nav-link" href="index.html">Home</a></li>
-          <li><a class="main-nav-link" href="about.html">About</a></li>
-          <li><a class="main-nav-link" href="services.html">Services</a></li>
-          <li><a class="main-nav-link" href="blog.html">Blog</a></li>
+          <li>
+            <a class="main-nav-link" href="./index.html">Acasa</a>
+          </li>
+          <li>
+            <a class="main-nav-link main-nav-link-active" href="./despre.html"
+              >Despre</a
+            >
+          </li>
+          <li><a class="main-nav-link" href="./servicii.html">Servicii</a></li>
+          <li><a class="main-nav-link" href="./echipa.html">Echipa</a></li>
+          <li><a class="main-nav-link" href="./galerie.html">Galerie</a></li>
+          <li>
+            <a class="main-nav-link nav-cta" href="blog.html"
+              >Fa o programare</a
```

Comment:
Same as above ☝️

Create a folder with this name (**programare**) and place inside this folder the corresponding HTML file, but rename it to `index.html`.

Reply:
This is fixed now